temba/channels/models.py

```diff
@@ -2013,10 +2013,8 @@ def send_vumi_message(cls, channel, msg, text):
         is_ussd = channel.channel_type in Channel.USSD_CHANNELS
         channel.config['transport_name'] = 'ussd_transport' if is_ussd else 'mtech_ng_smpp_transport'
 
-        if msg.response_to_id:
-            in_reply_to = Msg.objects.get(pk=msg.response_to_id).external_id
-        else:
-            in_reply_to = None
+        in_reply_to = Msg.objects.values_list(
+            'external_id', flat=True).get(pk=msg.response_to_id)
```

Comment:
@nicpottier @smn I just stumbled upon this and I don't think that's going to work, because values_list will return a QS with external_ids only, try Msg.objects.only('external_id') that will select only the external_id and pk columns. Also the .get() might throw a big exception here.The tests might not be complete enough if they haven't popped this up, unless I'm mistaken. I'm doing something very similar right now for the session and I moved this to a separate method so it's easier to test, looks like this:
```   
def get_session_with_status_only(self, session_id):         
        return self.only('status').filter(id=session_id).first()